Unit tests fail because webpack-stats.json don't exists

Hi,
I'm having problems running unit tests because `webpack-stats.json` don't exists in my CI, the file is not under version control along with the code. Do i need to add it? Is there a Test mode to avoid adding this file under VCS?
The feature requested in #137 could be a good solution, this makes possible creating a specific loader for testing purpose.

Tks,
Sandro Rodrigues

Creating a fake stat file for the tests didn't work for me, the default `WebpackLoader` raises an exception:

```
        if assets.get('status') == 'done':
            chunks = assets['chunks'].get(bundle_name, None)
            if chunks is None:
>               raise WebpackBundleLookupError('Cannot resolve bundle {0}.'.format(bundle_name))
E               webpack_loader.exceptions.WebpackBundleLookupError: Cannot resolve bundle project.
webpack_loader/loader.py:114: WebpackBundleLookupError
```

I created a `TestingWebpackLoader` as [per the example above](https://github.com/django-webpack/django-webpack-loader/issues/187#issuecomment-901449290) and it worked like a charm, thanks for the solution 👍🏻 

Would it make sense to include such a loader as part of this library to help users with testing?

--- a/README.md
+++ b/README.md
@@ -180,11 +180,11 @@ First we build the assets and, since we have `webpack-bundle-tracker` in our fro
 However, production usage for this package is **fairly flexible**. Other approaches may include keeping the production bundles in the version control and take that responsibility from the automatic pipeline. However, you must remember to always build the frontend and generate the bundle before pushing to remote.
 
 ## Usage in tests
-There are 2 approaches for when `render_bundle` shows up in tests, since we don't have `webpack-bundle-tracker` at that point to generate the stats file.
+To run tests where `render_bundle` shows up, since we don't have `webpack-bundle-tracker` at that point to generate the stats file, the calls to render the bundle will fail. The solution is to use the `FakeWebpackLoader` in your test settings:
 
-1. The first approach is to have specific settings for them (which is how we approach on our [tests](https://github.com/django-webpack/django-webpack-loader/blob/master/tests/app/settings.py#L111-L125)), such as done [here](https://github.com/django-webpack/django-webpack-loader/issues/187#issuecomment-470055769). Please note that it's necessary to have a pre-made stats file for the tests (which in general can be empty, such as [here](https://github.com/django-webpack/django-webpack-loader/issues/187#issuecomment-464250721)).
-
-2. The second approach is to leverage [`LOADER_CLASS` overriding](#extra-settings) for the test settings and customize the `get_bundle` method to return the url of a stats file. Note that, using this approach, the stats file doesn't have to [exist](https://github.com/django-webpack/django-webpack-loader/issues/187#issuecomment-901449290).
+```python
+WEBPACK_LOADER['DEFAULT']['LOADER_CLASS'] = 'webpack_loader.loader.FakeWebpackLoader'
+```
 
 ## Advanced Usage
 ### Rendering by file extension

--- a/webpack_loader/loader.py
+++ b/webpack_loader/loader.py
@@ -139,3 +139,19 @@ class WebpackLoader(object):
             "The stats file does not contain valid data. Make sure "
             "webpack-bundle-tracker plugin is enabled and try to run "
             "webpack again.")
+
+
+class FakeWebpackLoader(WebpackLoader):
+    """
+    A fake loader to help run Django tests.
+
+    For running tests where `render_bundle` is used but assets aren't built.
+    """
+
+    def get_bundle(self, _bundle_name):
+        return [
+            {
+                'name': 'test.bundle.js',
+                'url': 'http://localhost/static/bundles/test.bundle.js',
+            }
+        ]

